projects/plugins/jetpack/tests/e2e/lib/pages/homepage.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Internal dependencies
+ */
+import WpPage from './wp-page';
+import logger from '../logger';
+import { searchResultForTest1, searchResultForTest2 } from '../search-helper';
+
+export default class Homepage extends WpPage {
+	static SEARCH_API_PATTERN = /^https:\/\/public-api\.wordpress.com\/rest\/v1.3\/sites\/\d+\/search.*/;
+	static WAIT_FOR_ANIMATION_AND_RENDERING_TIMEOUT = 2000;
+
+	constructor( page ) {
+		const url = `${ siteUrl }/`;
+		super( page, { expectedSelectors: [ '.site-title' ], url } );
+	}
+
+	async registerRouteInterceptions() {
```

Comment:
I'm not sure what the purpose of this method. Can we call `searchAPIRoute` directly?

Reply:
fixed in https://github.com/Automattic/jetpack/pull/19447/commits/ddb8fae06203e91691943ca7e31b73d49975c87f